package.json

```diff
@@ -12,19 +12,19 @@
     "test": "test"
   },
   "dependencies": {
-    "d3-dsv": "^0.1.9"
+    "d3-dsv": "^1.0.3"
```

Comment:
I currently use PapaParse or BabyParse more often than d3-dsv.  Any reason to (or not to) substitute baby parse?  https://github.com/Rich-Harris/BabyParse